arcgis-ios-sdk-samples/Scenes/Choose camera controller/CameraControllerTableViewController.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+protocol CameraControllerTableViewControllerDelagate: AnyObject {
+    func selectedCameraController(_ cameraController: AGSCameraController)
```

Comment:
You're probably aware that by convention, delegate methods pass the object sending the message as the first parameter. In my experience, that parameter goes unused most of the time. However, it is the convention, so I try to follow it.

If you were to pass the camera controller picker, you could turn the `selectedRow` property into a `selectedCameraController` property and omit the `cameraController` parameter here. Then the delegate could get (and set) the selected camera controller directly instead of matching a camera controller to a row index. Just an idea.